src/main/java/seedu/address/model/company/Company.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.model.company;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Phone;
```

Comment:
I am not sure if we should reuse class under Person for Company? Not a big issue for now I suppose

Reply:
> If we work under the assumption that each Company will only have one person in charge, we can simply use a `Person` object.

Perhaps we could consider using UniquePersonList class and allow any number of liaises, but this is definitely no a high priority change since it is not essential to the core functionality of company (payroll purpose)